src/components/Alerts/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Alerts, AlertType, AlertWithIdentifier } from './Alerts';
```

Comment:
Maybe we should also export the Alert as well? This way you can integrate it outside of the Alerts container. For example within some documentation (and within the stories).

Reply:
I decided not to export the Alert on purpose. The only thing that would accomplish is that the user can use a single Alert outside of the AlertsContainer, which I don't see a useful usecase for. And all documentation should be derived from the Alerts component, which has all the types for the Alert